assets/js/googlesitekit/datastore/site/info.js

```diff
@@ -648,6 +660,36 @@ export const selectors = {
 	 */
 	getUpdateCoreURL: getSiteInfoProperty( 'updateCoreURL' ),
 
+	/**
+	 * Determines if the current user can update the Plugin.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(boolean|undefined)} `true` if the current user can update the plugin, otherwise `false`.
+	 */
+	getUpdatePluginCapacity: getSiteInfoProperty( 'updatePluginCapacity' ),
```

Comment:
This should be part of the user info, ideally part of `assets/js/googlesitekit/datastore/user/permissions.js`. I'm not sure why it was defined to use a different property for this capability but ideally this should be something we can select with `select( CORE_USER ).hasCapability( 'update_plugins' )`